servers.controllers.js

```diff
@@ -55,3 +56,19 @@ exports.getComments = (req, res, next) => {
     })
     .catch(next);
 };
+
+exports.postComments = (req, res, next) => {
+  const { username, body } = req.body;
+
+  const { article_id } = req.params;
+
+  checkIfArticleExists(article_id)
+    .then(() => {
+      return insertNewComments(username, body, article_id);
+    })
+    .then((result) => {
+      // console.log(result);
+      res.status(201).send(result);
```

Comment:
Please instead of result, can we have this on a more descriptive variable name? Also please send it back on an object with the descriptive name as the key.